src/types.ts

```diff
@@ -37,3 +37,10 @@ export interface AccessTokenCache {
   expires_in: number;
   created_on: number;
 }
+
```

Comment:
Can we call this Response instead of ResponseInfo? Also, where does Response come from? I don't see an import.

Reply:
It's coming from `lib.dom` https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L11771 That's enabled in tsconfig.json, so it's kinda global, that's why you don't see an import.

We can't name this type `Response` because of the collision with `Response` from `lib.dom`.

Reply:
Hmm, what about `ApiResponse`? Adding `Info` ironically adds no information.

Reply:
done